UE/Application/States/BaseState.cpp

```diff
@@ -100,4 +100,9 @@ void BaseState::handleTalkMessage(const std::string message)
     logger.logError("Unexpected handle talk message", message);
 }
 
+void BaseState::handleCallDrop(){
+    logger.logError("Unexpected handle calldrop message");
+
```

Comment:
Niepotrzebna spacja, ale to się usunie w innym commicie jak nic więcej do zmiany nie będzie.